test/jdk/java/awt/List/ListEnterExitTest.java

```diff
@@ -66,19 +70,23 @@ public void start() throws Exception {
             });
 
             final Robot robot = new Robot();
-            robot.delay(1000);
             robot.waitForIdle();
+            robot.delay(1000);
 
             EventQueue.invokeAndWait(() -> {
                 p = list.getLocationOnScreen();
             });
-            robot.mouseMove(p.x + 10, p.y + 10);
+            robot.mouseMove(p.x + 30, p.y + 40);
+            robot.waitForIdle();
             robot.delay(100);
+
+            robot.mouseMove(p.x - 30, p.y + 40);
             robot.waitForIdle();
-            robot.mouseMove(p.x - 10, p.y - 10);
             robot.delay(100);
+
+            robot.mouseMove(p.x + 30, p.y + 40);
             robot.waitForIdle();
-            robot.mouseMove(p.x + 10, p.y + 10);
+            robot.delay(100);
 
             robot.mousePress(InputEvent.BUTTON1_MASK);
             robot.mouseRelease(InputEvent.BUTTON1_MASK);
```

Comment:
Is this mouse press needed only to release `mouseEnterExitListener.wait`?

I suggest using a `CountDownLatch` to replace `MouseEnterExitListener.passed_1` and `passed_2`. The code would need to time-await for both latches to be released.

Then `MouseEnterExitListener.mousePressed` can be removed completely together with the synchronized block above.